assets/src/blocks/ENForm/ENForm.js

```diff
@@ -9,41 +9,37 @@ import {
   TextareaControl,
   ToggleControl,
   SelectControl,
-  ServerSideRender
+  ServerSideRender,
 } from '@wordpress/components';
-import {MediaPlaceholder} from "@wordpress/editor";
-import {ValidationMessage} from "../../components/enform/ValidationMessage/ValidationMessage";
+import {MediaPlaceholder} from '@wordpress/editor';
+import {ValidationMessage} from '../../components/enform/ValidationMessage/ValidationMessage';
 
 const {__} = wp.i18n;
 
 export class ENForm extends Component {
-  constructor(props) {
-    super(props);
-  }
-
-  isURLValid( url ) {
+  isURLValid(url) {
     return !!url ? url.substr(0, 8).toLowerCase() === 'https://' : true;
   }
 
   renderEdit() {
-    const {getCurrentPostType} = wp.data.select("core/editor");
-    const currentPostType      = getCurrentPostType();
+    const {getCurrentPostType} = wp.data.select('core/editor');
+    const currentPostType = getCurrentPostType();
 
     let flattenedPages = [];
     let pagesByType;
 
-    for (var i in window.p4en_vars.pages) {
-      pagesByType = window.p4en_vars.pages[i].map(page => {
-        return { label: page.name, value: page.id };
+    for (const i in window.p4en_vars.pages) {
```

Comment:
So because this is a `for-in-loop`, `i` is initialised for each iteration so it is no problem, if we were using an imperative for loop where `i` is incremented, then we will use `let`.